datacenter/dtr/2.0/configure/config-security.md

```diff
@@ -177,37 +177,17 @@ If you enter a "Domain Name" into the Security settings, it needs to be DNS
 resolvable on any client daemons that are running in `insecure-registry`
 mode.
 
-To set the flag, perform the following directions for your operating system.
-
-#### Ubuntu
-
-On Ubuntu 14.04 LTS, customize the Docker daemon configuration with the
-`/etc/defaults/docker` file.
-
-Open or create the `/etc/defaults/docker` file, and add the
-`--insecure-registry` flag to the `DOCKER_OPTS` setting (which may need to be
-added or uncommented) as follows:
-
-```
-DOCKER_OPTS="--insecure-registry dtr.yourdomain.com"
-```
-
-Then restart the Docker daemon with `sudo service docker restart`.
-
-#### RHEL/Centos
-
-On RHEL/Centos, customize the Docker daemon configuration with the
-`/etc/sysconfig/docker` file.
-
-Open or create the `/etc/sysconfig/docker` file, and add the
-`--insecure-registry` flag to the `OPTIONS` setting (which may need to be
-added or uncommented) as follows:
-
-```
-OPTIONS="--insecure-registry dtr.yourdomain.com"
+To set the flag, edit the `/etc/docker/daemon.json` file. If the file does not
+yet exist, create it. Assuming the file was empty, it should have the following
+contents:
+
+```json
+{
+  "insecure-registries" : ["dtr.yourdomain.com"]
```

Comment:
Stylistically, there generally isn't a space between the key and the `:` character.

```
"insecure-registries": ["dtr.yourdomain.com"]
```